lib/constable_web.ex

```diff
@@ -76,14 +75,12 @@ defmodule ConstableWeb do
       use Ecto.Schema
       import Ecto.Changeset
       import Ecto.Query
-      import Exgravatar
       @timestamps_opts [type: :utc_datetime, usec: false]
     end
   end
 
   def serializer do
```

Comment:
What would you think of removing `serializer` entirely? Please pardon my naïvety here, as this may be a required Phoenix protocol or something of that nature.

It looks like this is only in use in [User serializer](https://github.com/thoughtbot/constable/blob/620e583434b2dde2d0784adfdf7f65b64bf46e93/lib/constable/models/serializers/user.ex). From what I gather, it would simply unquote here and do nothing. Is it O.K. to also delete the line: `use ConstableWeb, :serializer` from the `User` serializer?

Reply:
I was actually hoping someone with more Phoenix experience would chime in! When I was looking at this I thought it should be fine to just remove it from here and remove the use in the user serializer, but I wasn't sure if there was a "right" thing to do in Phoenix. I'm inclined to remove it if we both think that's better.

Reply:
Well it looks like the recent change introduced `profile_provider/0`, so now `serializer/0` is here to stay :moyai: